kmk/handlers/stock.py

```diff
@@ -102,7 +102,7 @@ def bkdl_released(key, keyboard, KC, *args, **kwargs):
 
 
 def sleep_pressed(key, keyboard, KC, *args, **kwargs):
-    sleep_ms(key.meta.ms)
+    ticks_ms(key.meta.ms)
```

Comment:
this seems wrong, doesn't it still need to be sleep_ms (or `time.sleep(key.meta.ms/1000)`)?